src/data/showcases.js

```diff
@@ -1103,6 +1103,15 @@ export const Showcases = [
     source: null,
     tags: [ "analytics", "token", "minting"],
   },
+  {
+    title: "Shareslake",
+    description: 
+      "The stable branch of Cardano. A fiat-backed stablecoin for Cardano mainnet and a whole Cardano network running as a sidechain by the fiat-backed stablecoin instead of ADA. Deploy dApps out of the box and use Cardano technology like if you were using US dollars.",
+    preview: require("./showcase/shareslake.png"),
+    website: "https://www.shareslake.com/",
+    source: null,
+    tags: ["ecosystem", "gateway", "token"],
```

Comment:
```suggestion
    tags: ["gateway", "token"],
```
since @fill-the-fill passed this, he might correct me if I'm wrong, but the `ecosystem` tag... while matching a general description of this project as an "ecosystem"... isn't appropriate for how it's used here on the Developer Portal: which is for Showcase projects that describe _the Cardano ecosystem itself_ (i.e. it's not a category for "ecosystems").

Reply:
My bad, agree with you on this one